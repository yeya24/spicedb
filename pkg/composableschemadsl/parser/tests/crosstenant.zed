definition sometenant/somedef {
    relation somerel: anothertenant/someobject
}