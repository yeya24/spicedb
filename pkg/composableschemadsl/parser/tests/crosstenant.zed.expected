NodeTypeFile
  end-rune = 79
  input-source = cross tenant test
  start-rune = 0
  child-node =>
    NodeTypeDefinition
      definition-name = sometenant/somedef
      end-rune = 79
      input-source = cross tenant test
      start-rune = 0
      child-node =>
        NodeTypeRelation
          end-rune = 77
          input-source = cross tenant test
          relation-name = somerel
          start-rune = 36
          allowed-types =>
            NodeTypeTypeReference
              end-rune = 77
              input-source = cross tenant test
              start-rune = 54
              type-ref-type =>
                NodeTypeSpecificTypeReference
                  end-rune = 77
                  input-source = cross tenant test
                  start-rune = 54
                  type-name = anothertenant/someobject